### PR TITLE
Added links to admin panel

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Review by fightingtexasaggie — APPROVED** [review]
Tested, working. LGTM ;-)

File: src/Controller/Admin/DashboardController.php [hunk 1]
```diff
@@ -52,6 +52,10 @@ public function configureMenuItems(): iterable
         yield MenuItem::section('Editors');
         yield MenuItem::linkToCrud('IP Product Descriptor', 'fas fa-list-alt', InformationProductTypeDescriptor::class);
         yield MenuItem::linkToCrud('IP Digital Resource Descriptor', 'fas fa-list-alt', DigitalResourceTypeDescriptor::class);
+        yield MenuItem::section('Lists');
+        yield MenuItem::linkToUrl('Information Products', 'fas fa-list-alt', $this->generateUrl('pelagos_app_ui_information_products'));
```
Comment (mickel1138) [inline src/Controller/Admin/DashboardController.php:56]: Shall we use a different icon for this?

**Review by mickel1138 — APPROVED** [review]
The links on admin panel look good.

File: src/Controller/Admin/DashboardController.php (hunk 1 above)
Comment (praneethpr) [reply src/Controller/Admin/DashboardController.php:56]: I added a different icon for the Create New

**Pull request merged**